central/pruning/pruning.go

```diff
@@ -507,42 +511,50 @@ func (g *garbageCollectorImpl) markOrphanedAlertsAsResolved(deployments set.Froz
 	}
 }
 
-func isOrphanedDeployment(deployments set.FrozenStringSet, info *storage.NetworkEntityInfo) bool {
-	return info.GetType() == storage.NetworkEntityInfo_DEPLOYMENT && !deployments.Contains(info.GetId())
-}
+func (g *garbageCollectorImpl) removeOrphanedNetworkFlows(clusters set.FrozenStringSet) {
+	var wg sync.WaitGroup
+	sema := semaphore.NewWeighted(flowsSemaphoreWeight)
+
+	orphanTime := time.Now().Add(-1 * orphanWindow)
```

Comment:
So previously I had done something like in the other pruning, but I wonder if your way is just generally better
```
time < now() at time zone 'utc' - INTERVAL '%d MINUTES'
```